kahuna/public/js/edits/image-editor.html

```diff
@@ -64,11 +64,11 @@
                    ng:switch-when="ready"
                    ui:sref="image({imageId: ctrl.image.data.id})">View image ▸</a>
 
-                <span class="result-editor__status status status--invalid"
+                <a class="result-editor__status status status--invalid"
                       ng:switch-when="invalid">
                         Invalid
-                    <gr-icon title="You need to add both a credit and description.">help</gr-icon>
-                </span>
+                    <gr-icon title="This image will be uncroppable">help</gr-icon>
```

Comment:
The reasons are visible on the image page - there are now so many possible reasons that trying to squeeze an "invalid reasons list" into the UI here seems like a separate job.

The error message before was incorrect - in that the reasons it suggests for validity may not be true.